### PR TITLE
Skip validations for optional fields in the HAR spec that are not required for conversion to a collection

### PR DESCRIPTION
**Pull request by diyaeninan**

### CODE REVIEW

File: lib/schemas/HAR/timings.json [hunk 1]
```diff
@@ -7,18 +7,10 @@
     "receive"
   ],
   "properties": {
-    "dns": {
-      "type": "number",
-      "min": -1
-    },
     "connect": {
```
Comment (VShingala) [inline lib/schemas/HAR/timings.json]: In the specification, this is mentioned as optional and to be removed but it's not removed here.

File: lib/schemas/HAR/entry.json [hunk 2]
```diff
@@ -36,18 +36,6 @@
...
-    "serverIPAddress": {
-      "type": "string",
-      "oneOf": [
-        { "format": "ipv4" },
-        { "format": "ipv6" },
-        { "maxLength": 0 },
-        { "format": "ipv6brackets" }
-      ]
-    },
-    "connection": {
```
Comment (VShingala) [inline lib/schemas/HAR/entry.json:48]: This is not mentioned in the specification, is removal intended?

**Review by VShingala — COMMENTED** [review]
Added few comments on PR and specification.

File: lib/schemas/HAR/timings.json (hunk 1 above)
Comment (diyaeninan) [reply lib/schemas/HAR/timings.json]: Thanks, missed this! Will remove them

File: lib/schemas/HAR/entry.json (hunk 2 above)
Comment (diyaeninan) [reply lib/schemas/HAR/entry.json:48]: Yes, skipping `connection` is intended, will update the specification to reflect this

File: lib/schemas/HAR/postData.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "$id": "postData.json#",
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "type": "object",
+  "type": ["object", "null"],
   "optional": true,
   "required": [
```
Comment (VShingala) [inline lib/schemas/HAR/postData.json:6]: There is `text` field which is not mentioned as required but it also does not contain type `null`. Is it intended?

**Review by VShingala — COMMENTED** [review]
Rest looks good apart from one comment.

File: lib/schemas/HAR/postData.json (hunk 3 above)
Comment (diyaeninan) [reply lib/schemas/HAR/postData.json:6]: `text` in postData is not marked as optional in http://www.softwareishard.com/blog/har-12-spec/#postData

Comment (VShingala) [reply lib/schemas/HAR/postData.json:6]: Got it, the shouldn't it be described under `required` array?

Comment (diyaeninan) [reply lib/schemas/HAR/postData.json:6]: I will add a `null` option here as well, on looking at the har schemas defined [here](https://github.com/ahmadnassri/har-schema/blob/master/lib/postData.json) and [here](http://www.softwareishard.com/har/viewer/) text seems optional as well, unlike the description

**Review by VShingala — APPROVED** [review]
LGTM!

**Pull request merged**